js/components/dataset/list.vue

```diff
@@ -21,7 +21,8 @@ export default {
                 key: 'title',
                 sort: 'title',
                 align: 'left',
-                type: 'text'
+                type: 'text',
+                width: 200
```

Comment:
But I agree with @davidbgk : we shouldn't set a width on the column that should take all the empty space. Most of the time, titles are very long and span on 2 lines, with a fixed width on 200, they will span more.